eyeball-im/src/vector.rs

```diff
@@ -196,6 +196,20 @@ impl<T: Clone + Send + Sync + 'static> ObservableVector<T> {
         }
     }
 
+    /// Truncate the vector to `len` elements and notify subscribers.
+    ///
+    /// Does nothing if `len` is greater or equal to the vector's current
+    /// length.
```

Comment:
Usually it panics, but I'm fine with this behavior too.

Reply:
That's just for imbl, std does the same thing as eyeball-im: https://github.com/jneem/imbl/issues/66